roles/tinc-dynamic/defaults/main.yml

```diff
@@ -1,12 +1,7 @@
 ---
-
-vpn_name: idbvpn
-vpn_name2: testvpn
-vpn_interface: "{{ vpn_name2 }}"
-
 # unset to disable logging
 vpn_log_level: 1
 
 # if vpn_log_level is set but vpn_log_file is not,
 # it will use syslog by default.
-vpn_log_file: "/var/log/tinc.{{ vpn_name2 }}.log"
+vpn_log_file: "/var/log/tinc.{{ vpn_name }}.log"
```

Comment:
Dont provide a path to `--logfile`.

See https://github.com/ideascube/ansiblecube/commit/01e68b8804e234637bffd3acaf82d7d7206bc91f